src/main/java/io/github/thebusybiscuit/slimefun4/core/attributes/DamageableItem.java

```diff
@@ -49,7 +49,7 @@ default void damageItem(@Nonnull Player p, @Nullable ItemStack item) {
         if (isDamageable() && item != null && !item.getType().isAir() && item.getAmount() > 0) {
             int unbreakingLevel = item.getEnchantmentLevel(Enchantment.DURABILITY);
 
-            if (unbreakingLevel > 0 && Math.random() * 100 <= (60 + Math.floorDiv(40, (unbreakingLevel + 1)))) {
+            if (unbreakingLevel > 0 && !(Math.random() < (1.0 / (unbreakingLevel + 1)))) {
```

Comment:
Now this method is gonna break with any armor items though.
You should probably extract this statement into a dedicated method, so it can be overridden and then provide two constant formaulas for it in our utils package, where it will default to the tools Formula.